cmd/machine-config-operator/bootstrap.go

```diff
@@ -51,20 +47,12 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.pullSecretFile, "pull-secret", "/assets/manifests/pull.json", "path to secret manifest that contains pull secret.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.destinationDir, "dest-dir", "", "The destination directory where MCO writes the manifests.")
 	bootstrapCmd.MarkFlagRequired("dest-dir")
-	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.mccImage, "machine-config-controller-image", "", "Image for Machine Config Controller.")
-	bootstrapCmd.MarkFlagRequired("machine-config-controller-image")
-	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.mcsImage, "machine-config-server-image", "", "Image for Machine Config Server.")
-	bootstrapCmd.MarkFlagRequired("machine-config-server-image")
-	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.mcdImage, "machine-config-daemon-image", "", "Image for Machine Config Daemon.")
-	bootstrapCmd.MarkFlagRequired("machine-config-daemon-image")
```

Comment:
we need an intermediate PR to remove the MarkFlagRequired first from all these above. Merge that and then merge https://github.com/openshift/installer/pull/1875 and then back again here to basically merge this PR. But we need to drop the required flags to merge the installer PR

Reply:
AHH gotcha thanks!